components/GameForm.jsx

```diff
@@ -0,0 +1,56 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+export default function GameForm({ onSubmit }) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onSubmit(data);
+  }
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <label htmlFor="gameTitleInput">Title:</label>
+      <input type="text" id="gameTitleInput" name="title" required></input>
+      <label htmlFor="gameRatingInput">Your rating:</label>
+      <input
+        type="range"
+        min="0"
+        max="10"
+        step="1"
+        id="gameRatingInput"
+        name="rating"
+        required
+      ></input>
+      <label htmlFor="gameDescriptionInput">Description:</label>
+      <textarea
+        rows="8"
+        placeholder="Your description"
+        id="gameDescriptionInput"
+        name="description"
+        required
+      />
+      <StyledButtonRow>
+        <button type="submit">Create</button>
+        <button onClick={() => router.push("/")}>Cancel</button>
```

Comment:
You might want to switch the positions of those two button.

When I tried to add a new game, it felt counterintuitive to have the create button on the left and the cancel button on the right.

How do you feel about this? If you agree with me, I recommend simply switching the buttons.